src/note.cpp

```diff
@@ -493,7 +506,7 @@ void Note::UpdateFromTransPitch(TransPitch tp)
     }
 
     if (this->GetOct() != tp.m_oct) {
-        if(this->HasOctGes()) {
+        if (this->HasOctGes()) {
```

Comment:
Just checking: do you have and/or know about clang-format, which will fix these sorts of problems automatically: https://electronjs.org/docs/development/clang-format

Usage:

```bash
clang-format -i file.cpp
```

where the `-i` option will edit the file in place.

Reply:
Yeah, I have a git pre-commit hook set up to run this for me over files that I'm committing. (but I believe clang-format doesn't give a different return value when it succeeds, so my pre-commit hook doesn't remind me to stage changes that it made).

Reply:
(I guess I should be using `git-clang-format`)